doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -160,61 +318,91 @@ Enabling partition mode
+ACRN hypervisor, you should see the output from the ACRN serial console as below.
+You could then log in to the privileged VMs by ``sos_console`` command,
+and press CTRL-Space keys to return to the ACRN serial console.
+
+.. code-block:: console
+  :emphasize-lines: 14,31
+
+  ACRN Hypervisor
+  calibrate_tsc, tsc_khz=1094400
+  [21017289us][cpu=0][sev=2][seq=1]:HV version 0.3-unstable-2018-11-08 12:41:24-ef974d1a-dirty DBG (daily tag:acrn-2018w45.3-140000p) build by clear, start time 20997424us
+  [21034127us][cpu=0][sev=2][seq=2]:API version 1.0
+  [21039218us][cpu=0][sev=2][seq=3]:Detect processor: Intel(R) Pentium(R) CPU N4200 @ 1.10GHz
+  [21048422us][cpu=0][sev=2][seq=4]:hardware support HV
+  [21053897us][cpu=0][sev=1][seq=5]:SECURITY WARNING!!!!!!
+  [21059672us][cpu=0][sev=1][seq=6]:Please apply the latest CPU uCode patch!
+  [21074487us][cpu=0][sev=2][seq=28]:Start VM1
+  [21074488us][cpu=3][sev=2][seq=29]:Start VM2
+  [21885195us][cpu=0][sev=3][seq=34]:vlapic: Start Secondary VCPU1 for VM[1]...
+  [21889889us][cpu=3][sev=3][seq=35]:vlapic: Start Secondary VCPU1 for VM[2]...
+  ACRN:\>
+  ACRN:\>sos_console 1
+
+  ----- Entering Guest 1 Shell -----
+  [    1.997439] systemd[1]: Listening on Network Service Netlink Socket.
+  [  OK  ] Listening on Network Service Netlink Socket.
+  [    1.999347] systemd[1]: Created slice system-serial\x2dgetty.slice.
+  [  OK  ] Created slice system-serial\x2dgetty.slice.
+  [  OK  ] Listening on Journal Socket (/dev/log).
+  ...
+  clr-932c8a3012ec4dc6af53790b7afbf6ba login: root
+  Password: 
+  root@clr-932c8a3012ec4dc6af53790b7afbf6ba ~ # lspci
+  00:00.0 Host bridge: Intel Corporation Celeron N3350/Pentium N4200/Atom E3900 Series Host Bridge (rev 0b)
+  00:01.0 SATA controller: Intel Corporation Celeron N3350/Pentium N4200/Atom E3900 Series SATA AHCI Controller (rev 0b)
+  root@clr-932c8a3012ec4dc6af53790b7afbf6ba ~ # 
+
+   ---Entering ACRN SHELL---
+  ACRN:\>sos_console 2
+
+  ----- Entering Guest 2 Shell -----
+  [    1.490122] usb 1-4: new full-speed USB device number 2 using xhci_hcd
+  [    1.621311] usb 1-4: not running at top speed; connect to a high speed hub
+  [    1.627824] usb 1-4: New USB device found, idVendor=058f, idProduct=6387, bcdDevice= 1.01
+  [    1.628438] usb 1-4: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+  ...
+  clr-2e8082cd4fc24d57a3c2d3db43368d36 login: root
+  Password: 
+  root@clr-2e8082cd4fc24d57a3c2d3db43368d36 ~ # lspci
+  00:00.0 Host bridge: Intel Corporation Celeron N3350/Pentium N4200/Atom E3900 Series Host Bridge (rev 0b)
+  00:01.0 USB controller: Intel Corporation Celeron N3350/Pentium N4200/Atom E3900 Series USB xHCI (rev 0b)
+  00:02.0 Ethernet controller: Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller (rev 0c)
+  root@clr-2e8082cd4fc24d57a3c2d3db43368d36 ~ #
+
+.. [1] At the time of writing, the ACRN kernel source at the commit ``afc22d8``,
```

Comment:
If these commit IDs are very important to know (e.g., the current master branch might not work), this footnote should be shown as a ``.. note::`` directive much earlier in the doc.

Reply:
@dbkinder In theory, commits after these points should work unless there is a regression. I think these commits are just known-to-work points, they are valuable only when the user hit the regression. How do you want to position it?

Reply:
If the master branch should work, given your comment, I'd leave off the footnote talking about the commit IDs. If a problem shows up, it's a bug and there's no reason for a developer to go to a previous commit (and lose whatever good things happened after that).

Reply:
Can't agree you more. Will remove the footnote and references to it.